networking/configuring-cluster-wide-proxy.adoc

```diff
@@ -15,6 +15,22 @@ You can configure a cluster-wide proxy during cluster installation.
 
 * You are the cluster owner.
 * Your account has sufficient privileges.
+* You have an existing `install-config.yaml` file.
+* You reviewed the sites that your cluster requires access to and determined whether any of them need to bypass the proxy. By default, all cluster egress traffic is proxied, including calls to hosting cloud provider APIs. You added sites to the `Proxy` object's `spec.noProxy` field to bypass the proxy if necessary.
++
```

Comment:
These lines can all be removed - they're not relevant for the managed service offering.

Reply:
@mrbarge I removed lines 18 and 19. Are we good to merge this?

Reply:
Thank you! This looks fine to me.